.github/workflows/snap-ci.yaml

```
@@ -0,0 +1,89 @@
+name: Snap CI
+
+on:
+  schedule:
+    - cron: "20 2 * * 1" # Monday mornings 02:20 UTC
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  # Allow manual trigger
+  workflow_dispatch:
+
+env:
+  SNAP_NAME: chip-tool
+
+jobs:
+  matrix-build:
```

```suggestion
  build:
```
Mixing keywords and names causes confusion.